_data/glossary.yaml

```diff
@@ -8,11 +8,6 @@ aufs: |
 base image: |
   A **base image** has no parent image specified in its Dockerfile. It is created
   using a Dockerfile with the `FROM scratch` directive.
-boot2docker: |
```

Comment:
Wondering if we should keep a placeholder, and describe it's no longer a thing

Guess Google will still give them results, so perhaps not a problem